public/scss/navigation.scss

```diff
@@ -126,16 +126,32 @@ ul.pages > li:last-child > a {
 }
 
 .blocks-selector.active {
-  background-color: #ffd16a;
+  background-color: #2ed6a1;
 }
 
-select.dropdown {
+select {
+  -webkit-appearance: none;
+  -moz-appearance: none;
   border: none;
+  height: 30px;
   font-size: 14px;
-  height: 24px;
   padding: 0 5px;
+  border-radius: 2px;
+  background-color: #fff !important;
```

Comment:
Doesn't work for dark mode.

![image](https://user-images.githubusercontent.com/6109680/66825193-633cfd00-ef0f-11e9-87a4-7eb8b6034ef4.png)

Also, why the `!important`?